docs/tools/tikv-control.md

```diff
@@ -255,3 +257,64 @@ success!
 > - This command only supports the local mode. It prints `success!` when successfully run.
 > - The argument of the `-p` option specifies the PD endpoints without the `http` prefix. Specifying the PD endpoints is to query whether the specified `region_id` is validated or not.
 > - You need to run this command for all stores where specified Regions' peers are located.
+
+### Ldb Cmd
+
+The ldb command line tool offers multiple data access and database admin commands. Some examples are listed below. 
+For more information, please consult the help message displayed when running `tikv-ctl ldb` without any arguments.
+
+Example data access sequence:
+```bash
+$ tikv-ctl ldb --db=/tmp/test_db --create_if_missing put a1 b1
+OK 
+$ tikv-ctl ldb --db=/tmp/test_db get a1
+b1
+$ tikv-ctl ldb --db=/tmp/test_db get a2
+Failed: NotFound:
+$ tikv-ctl ldb --db=/tmp/test_db scan
+a1 : b1
+$ tikv-ctl ldb --db=/tmp/test_db scan --hex
+0x6131 : 0x6231
+$ tikv-ctl ldb --db=/tmp/test_db put --key_hex 0x6132 b2
```

Comment:
Is it proper to guide user modify the data? Users may use this tool to manipulate the data of TiKV.

Reply:
This is a good point, the tool is actually quite dangerous. I think the examples here will be more useful if we can show some usual cases, like `dump` or `dump_manifest`, instead of some other commands that we rarely use.

Reply:
I think we can just keep some "dump" related commands here. If we need to use other commands, just check the help message or RocksDB documents.